sfge/src/main/java/com/salesforce/graph/ops/MethodUtil.java

```diff
@@ -165,24 +165,30 @@ private static void addMessagesForTarget(RuleRunnerTarget target, List<MethodVer
      */
     public static List<MethodVertex> getAuraEnabledMethods(
             GraphTraversalSource g, List<String> targetFiles) {
-        // Only look at UserClass vertices. Not interested in Enums, Interfaces, or Triggers
+        return getMethodsWithAnnotation(g, targetFiles, Schema.AURA_ENABLED);
+    }
+
+    /**
+     * Returns non-test methods in the target files with a @NamespaceAccessible annotation. An empty
+     * list implicitly includes all files.
+     */
+    public static List<MethodVertex> getNamespaceAccessibleMethods(
+            GraphTraversalSource g, List<String> targetFiles) {
+        return getMethodsWithAnnotation(g, targetFiles, Schema.NAMESPACE_ACCESSIBLE);
+    }
+
+    static List<MethodVertex> getMethodsWithAnnotation(
```

Comment:
Package-private instead of true-private, so that we can have one parameterized test for this method instead of a bunch of tests for each invocation of it.